guides/getting_started_extension.md

```diff
@@ -4,32 +4,60 @@ title: "Getting Started with Extensions"
+This will archive this Extension and pass it as an asset to the Cosmos CDK Toolkit s3 bucket in your master account, and trigger the CodeBuild job to bootstrap your Extension. You can watch the CodeBuild project logs (cdk-toolkit-bootstrap-project) and CloudFormation to see what resources are being created.
 
-`git remote set-url origin "https://git-codecommit.your-region.amazonaws.com/v1/repos/app-your-project-name-cdk-repo"`
+A CodeCommit repository to house this newly customised Extension was created as part of the bootstrapping process above. Update the git repository in this Extension to point to the new CodeCommit respository. Replacing the _your-region_ section with the region you selected in Step 3 and the _your-app_ section with the name you gave your project in `bin/main.ts `, run the following command:
+
+    git remote set-url origin "https://git-codecommit.your-region.amazonaws.com/v1/repos/app-your-app-cdk-repo"
+
+You can also find the repository url in CodeCommit. Then:
 
 - Add the changes made to this template by running `git add .`
 - Commit the changes by running `git commit -m "inital commit"` 
 - Push the changes to CodeCommit by running `git push`
 
 Your Extension now has a CDK & Code Pipeline. Any further changes may be deployed using the Extension's own CDK Pipeline.
 
-## [Next Steps: Add Features]()
+### Solar Systems
+As with core, the Solar Systems need to be deployed after bootstrapping as they are dependent on Cosmos and Galaxy resources created on the initial run of the pipeline. To deploy, uncomment the solar system, push up your changes and re-run the App CDK pipeline. 
+
+> If you get an error 'AppCosmosSandboxDevDevSolarSystem failed: Error [ValidationError]: Unable to fetch parameters [/App/CosmosSandbox/Dev/Dev/VersionState] from parameter store for this account.' when running the cdk pipeline to add the solar systems, run the code pipeline first.
+
+#### Deploy Stages
+This template includes a separate deploy stage in the code pipeline for each new solar system. The below code would deploy the `dev` and `tst` stacks.
+
+    ciCd.addCdkDeployEnvStageToCodePipeline({
+        name: 'DeployDev',
+        stacks: [dev, tst],
+    });
+
+This is optional. The template includes a final deploy stage that will deploy all stacks, however, separating out stacks allows you to retry sections - for example, you may wish to run the dev stack to test your changes before approving a roll out to prod. It is also recommended to include a CiCd stage at the start of the pipeline, to minimise additional runs of the pipeline. You can add stages to both the Code pipeline (`ciCd.addCdkDeployEnvStageToCodePipeline`) and the CDK pipeline (`ciCd.addCdkDeployEnvStageToCdkPipeline`) for your extension.
```

Comment:
does addCdkDeployEnvStageToCdkPipeline exist on the current template? I added it manually for cafe

Reply:
ah, good point. it exists in the blank template extension, but not in ecr